filter_linux.go

```diff
@@ -1039,3 +1066,42 @@ func SerializeRtab(rtab [256]uint32) []byte {
 	_ = binary.Write(&w, native, rtab)
 	return w.Bytes()
 }
+
+func deserializeSockFilter(opsLen uint16, ops []byte) ([]SockFilter, error) {
+	if excp := int(opsLen) * 8; len(ops) != excp {
```

Comment:
Why do we need to define this `excp` inside this `if`?
I do not think it gives us much compared to 
```
if len(ops) != 8*opsLen {
           [...]", len(ops), 8*opsLen)
}
```